src/views/OrderDetail.vue

```diff
@@ -129,11 +129,11 @@ export default defineComponent({
     },
     markBopisItem (item: any, event: any) {
       if(this.isBopisItem(item)){
-        item.properties = item.properties.filter((property: any) => !(property.name === '_pickupstore' || property.name === 'Pickup Store'))
+        item.properties = item.properties.filter((property: any) => !(property.name === '_pickupstore' || property.name === 'Store Pickup' || property.name === 'Pickup Store'))
```

Comment:
Add a comment that added the check `Pickup Sotre` for backward compatibility.

Reply:
Added comment.